Formula/gcc-arm-embedded.rb

```diff
@@ -5,21 +5,76 @@ class GccArmEmbedded < Formula
   version "9-2019-q4-major"
   sha256 "f162a655f222319f75862d7aba9ff8a4a86f752392e4f3c5d9ef2ee8bc13be58"
 
-  # build-prerequisites fails without this, when ./configure is run from zlib it raises an error,
-  # the zlib log shows the problem is: "The build tool has reset ENV; --env=std required."
-  env :std
-
   depends_on "gnu-tar" => :build
   depends_on :xcode => ["10.0", :build]
 
+  resource "gmp" do
+    url "https://gmplib.org/download/gmp/gmp-6.1.0.tar.bz2"
+    sha256 "498449a994efeba527885c10405993427995d3f86b8768d8cdf8d9dd7c6b73e8"
+  end
+
+  resource "mpfr" do
+    url "https://www.mpfr.org/mpfr-3.1.4/mpfr-3.1.4.tar.bz2"
+    sha256 "d3103a80cdad2407ed581f3618c4bed04e0c92d1cf771a65ead662cc397f7775"
+  end
+
+  resource "mpc" do
+    url "ftp://ftp.gnu.org/gnu/mpc/mpc-1.0.3.tar.gz"
+    sha256 "617decc6ea09889fb08ede330917a00b16809b8db88c29c31bfbb49cbf88ecc3"
+  end
+
+  resource "isl" do
+    url "ftp://gcc.gnu.org/pub/gcc/infrastructure/isl-0.18.tar.bz2"
+    sha256 "6b8b0fd7f81d0a957beb3679c81bbb34ccc7568d5682844d8924424a0dadcb1b"
+  end
+
+  resource "expat" do
+    url "https://downloads.sourceforge.net/project/expat/expat/2.1.1/expat-2.1.1.tar.bz2"
+    sha256 "aff584e5a2f759dcfc6d48671e9529f6afe1e30b0cd6a4cec200cbe3f793de67"
+  end
+
+  resource "libelf" do
+    url "https://ftp.osuosl.org/pub/blfs/conglomeration/libelf/libelf-0.8.13.tar.gz"
+    mirror "https://fossies.org/linux/misc/old/libelf-0.8.13.tar.gz"
+    sha256 "591a9b4ec81c1f2042a97aa60564e0cb79d041c52faa7416acb38bc95bd2c76d"
+  end
+
+  resource "libiconv" do
+    url "https://ftp.gnu.org/pub/gnu/libiconv/libiconv-1.15.tar.gz"
+    sha256 "ccf536620a45458d26ba83887a983b96827001e92a13847b45e4925cc8913178"
+  end
+
+  resource "zlib" do
+    url "https://www.zlib.net/fossils/zlib-1.2.8.tar.gz"
+    sha256 "36658cb768a54c1d4dec43c3116c27ed893e88b02ecfcb44f2166f9c0b7f2a0d"
+  end
+
   def install
-    inreplace "build-common.sh", "JOBS=1", "JOBS=`sysctl -n hw.ncpu`"
+    ENV.deparallelize
+
+    # `clean_env` removes most ENV variables as a precaution. This can't happen if we want to use Superenv.
+    inreplace "build-common.sh", /clean_env$/, ""
+
+    cd "src" do
+      Dir.each_child(".") do |f|
+        system "tar", "xf", f
+      end
+    end
+
+    resource("gmp").stage(buildpath/"src/gmp-6.1.0")
+    resource("mpfr").stage(buildpath/"src/mpfr-3.1.4")
+    resource("mpc").stage(buildpath/"src/mpc-1.0.3")
+    resource("isl").stage(buildpath/"src/isl-0.18")
+    resource("expat").stage(buildpath/"src/expat-2.1.1")
+    resource("libelf").stage(buildpath/"src/libelf-0.8.13")
+    resource("libiconv").stage(buildpath/"src/libiconv-1.15")
+    resource("zlib").stage(buildpath/"src/zlib-1.2.8")
```

Comment:
Is there no way to link these against the homebrew libraries?

Reply:
I'm not sure. Is there a way for me to link against the specific versions?

Reply:
No, and if it's not building with the latest versions this formula might not be a good fit for homebrew-core.

Reply:
GCC should build against the latest versions, at least for GMP, MPFR, MPC, and ISL (which are host libraries).

Reply:
Alright I'll see what I can do.